src/sentry/static/sentry/app/views/projectFilters.jsx

```diff
@@ -409,6 +414,20 @@ const ProjectFilters = React.createClass({
         });
       }
     });
+
+    this.api.request(`/projects/${orgId}/${projectId}/tombstone/`, {
+      method: 'GET',
+      success: data => {
```

Comment:
You don't need to make this change, but I want to point out that you can do the following (using the [Object Literal Property Value Shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015)):

```javascript
success: tombstones => { this.setState({tombstones}); }
```